webbpsf/opds.py

```diff
@@ -95,8 +93,7 @@ class OPD(poppy.FITSOpticalElement):
 
     """
 
-    def __init__(self, name='unnamed OPD', opd=None, opd_index=0, transmission=None,
-                 segment_mask_file=None, npix=1024, **kwargs):
+    def __init__(self, name="unnamed OPD", opd=None, opd_index=0, transmission=None, segment_mask_file=None, npix=1024, **kwargs):
```

Comment:
it's super not obvious to me how ruff is deciding where to split lines or not. Compare this edit, where it compresses two lines into one, versus the next...